tower-rate-limit/src/lib.rs

```diff
@@ -45,7 +44,7 @@ pub struct ResponseFuture<T> {
 #[derive(Debug)]
 enum State {
     // The service has hit its limit
-    Limited(Sleep),
+    Limited(Delay),
```

Comment:
Updated to use the new `tokio-timer` API as this crate no longer compiles otherwise. This is an unrelated change.